BE/Business.Access.Layer/Services/UserService.cs

```diff
@@ -87,8 +87,17 @@ private void CheckPassword(string password)
 
         private async Task CheckEmail(string email)
         {
```

Comment:
This method has no validation for special characters. Username part of the email should also have a null or white space check.